Modules/System/Time Zone Management/src/DateTimeOffsetImpl.Codeunit.al

```diff
@@ -0,0 +1,37 @@
+codeunit 8721 "DateTime Offset Impl."
+{
+    Access = Internal;
+
+    procedure GetUtcOffset(SourceDateTime: DateTime; TimeZoneId: Text): Duration
+    var
+        TimeZoneInfoInitializer: Codeunit "TimeZoneInfo Initializer";
+        Offset: Duration;
+        TimeZoneInfoDotNet: DotNet TimeZoneInfo;
```

Comment:
Looks like we need a dependency to the dotnet module.

Reply:
Added it.